src/components/BarChartCard/BarChartCard.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { Fragment } from 'react';
+import SimpleBarChart from '@carbon/charts-react/bar-chart-simple';
+import StackedBarChart from '@carbon/charts-react/bar-chart-stacked';
+import GroupedBarChart from '@carbon/charts-react/bar-chart-grouped';
+import '@carbon/charts/styles.css';
```

Comment:
do we not include this somewhere already? All styles should be brought into our main sass partial. @scottdickerson I see we are bringing this in TimeSeriesCard like this as well. Do they not offer a sass partial for each chart?

Reply:
there are no partials for the single charts, but I moved this out and created a globals/_charts to import this + having common things for the charts, like RTL behavior